app/models/card.rb

```diff
@@ -1,9 +1,17 @@
 class Card < ActiveRecord::Base
-  before_create :record_review_date
+  before_create  :record_review_date
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.